<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>1400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Button_detect">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>310</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>检测</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_exit">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>430</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QSlider" name="arg_Slider">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>500</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_imgcut">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>250</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>检测区域</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_save">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>370</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_result">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>60</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_result">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>总数</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_imginput">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>190</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>图片导入</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>130</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>拍照</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_show">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>720</width>
      <height>1280</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>720</width>
      <height>1280</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>718</width>
       <height>1278</height>
      </rect>
     </property>
    </widget>
   </widget>
   <zorder>label_show</zorder>
   <zorder>Button_detect</zorder>
   <zorder>Button_exit</zorder>
   <zorder>arg_Slider</zorder>
   <zorder>toolButton_imgcut</zorder>
   <zorder>Button_save</zorder>
   <zorder>lcdNumber_result</zorder>
   <zorder>label_result</zorder>
   <zorder>Button_imginput</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>scrollArea</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
